src/argilla/client/feedback/training/frameworks/trl.py

```diff
@@ -78,34 +118,57 @@ def __init__(
             (
                 TrainingTaskForSFT,
                 TrainingTaskForRM,
+                TrainingTaskForPPO,
                 TrainingTaskForDPO,
             ),
         ):
             raise NotImplementedError(f"Task {self._task} not supported in TRL.")
 
-        from trl import DPOTrainer, RewardTrainer, SFTTrainer
+        from trl import DPOTrainer, PPOTrainer, RewardTrainer, SFTTrainer
 
         self.trainer_mapping = {
             TrainingTaskForSFT: SFTTrainer,
             TrainingTaskForRM: RewardTrainer,
+            TrainingTaskForPPO: PPOTrainer,
             TrainingTaskForDPO: DPOTrainer,
         }
         self.trainer_cls = self.trainer_mapping[type(self._task)]
 
         self.init_training_args()
 
+        if isinstance(self._task, TrainingTaskForPPO):
+            self._logger.warning(
+                "The PPOTrainer must be initialized by passing `reward_model`, `length_sampler_kwargs`, `generation_kwargs` as kwargs to the `update_config()`-method."
+            )
+
```

Comment:
Is this not duplicate? We also send this warning in `init_training_args` on the previous line

Reply:
Yes, I think we want a logger to warn during `update_arguments` and an explicit raised exception during `train()`

Reply:
Sounds good, I'll add that